packages/core/src/App/Containers/Layout/header/header.tsx

```diff
@@ -86,11 +91,13 @@ const Header = observer(() => {
             case pathname === routes.onboarding:
                 result = null;
                 break;
+            case is_dtrader_v2_enabled && is_mobile && pathname === routes.trade:
```

Comment:
```suggestion
            case is_dtrader_v2_enabled && is_mobile && pathname.startsWith(routes.trade):
```
Now we have the subpaths like /dtrader/markets, so we should just check for the route which starts with /dtrader

Reply:
fixed thanks